tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -108,6 +97,53 @@ def run(**kw):
             f"{kw.get('ceph_cluster_dict').values} has less or more clusters Than Expected(2 clusters expected)"
         )
         log.exception(ve)
+        return 1
     except Exception as e:
         log.exception(e)
         return 1
+
+
+def run(**kw):
+    """
+    Snapshot based rbd mirroring schedulers at cluster and pool level.
+
+    Args:
+        **kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-83575375 and CEPH-83575376
+
+    Test Case Flow:
+    1. Configure snapshot based mirroring between two clusters
+    2. Add mirror snapshot schedule at cluster and pool level
+    3. create some images and wait for them to mirror to secondary
+    4. Check that mirror snapshots are created for each images
+    5. Create a snapshot mirror schedule and list the snapshots
+    6. View the status of schedule snapshot
+    7. remove the snapshot schedule from cluster and pool level, verify the same
+    8. Perform test steps for both Replicated and EC pool
+    """
+    log.info(
+        "Starting CEPH-83575375 and CEPH-83575376, \
```

Comment:
one small change, 
can you please remove this \ , its creating tab (space) in log.

Reply:
Done thanks.